sdk/scheduler/src/main/java/com/mesosphere/sdk/offer/NamedVIPRequirement.java

```diff
@@ -50,7 +50,7 @@ public OfferEvaluationStage getEvaluationStage(String taskName) {
         return new NamedVIPEvaluationStage(
                 getResource(),
                 taskName,
-                getPortName(),
+                getEnvName(),
```

Comment:
Maybe keep the same name from specs -- `envKey` -- to make this more consistent.